AdobeIms/Controller/Adminhtml/User/Profile.php

```diff
@@ -12,6 +12,7 @@
 use Magento\Backend\App\Action;
 use Magento\Framework\Controller\ResultFactory;
 use Psr\Log\LoggerInterface;
+use Magento\AdobeStockClientApi\Api\ClientInterface;
```

Comment:
Unfortunately, we cannot use adobe stock modules from `AdobeIms` as it has to be a separate independent package.

I think the solution can be: 
 - moving User data generation out of Controller to a dedicated model and applying a plugin to this model from AdobeStockAsset module.
OR
 - updating the quota via additional asynchronous request after login (this is more preferable than plugin) - additional request should not affect user experience as it should be performed only once after login

Reply:
ok